### DISCUSSION TITLE
ReferenceError __DEV__ is not defined

### DISCUSSION THREAD

**Discussion by Njinou** [codesandbox/codesandbox-client#4017] [General]
Hello, i have tried to use "react-native-maps": "0.27.1" but i am getting an error mentioned above.
I tried to define  a global DEV in the app.js but i am getting an error __DEV__ create but never used.
So i would like to know how to create a dev environment or maybe how to integrate "react-native-maps": "0.27.1".
Thanks for your help

**Comment by garethx** [comment]
Please share a link to the sandbox you're having issues with.
